source/_integrations/zwave_js.markdown

```diff
@@ -1008,32 +1008,48 @@ You can switch between the official Z-Wave JS add-on and the Z-Wave JS UI add-on
 
 ### How to switch from Z-Wave JS to the Z-Wave JS UI add-on?
 
-You can switch from the official **Z-Wave JS** add-on to the **Z-Wave JS UI** add-on. However, you cannot run them both at the same time. Only one of the add-ons can be active at the same time.
+You can switch from the official **Z-Wave JS** add-on to the community **Z-Wave JS UI** add-on. However, you cannot run them both at the same time. Only one of the add-ons can be active at the same time.
 
-Switching does not require renaming your devices.
+All of your HA devices and entities will remain unchanged if you follow these steps:
```

Comment:
We don't use HA as an abbreviation for Home Assistant.

If it needs to be clarified what devices and entities we are referring to, I'd write Z-Wave devices and entities, as that's the name of the integration. But I'm not sure that is needed.